debezium-core/src/main/java/io/debezium/relational/Column.java

```diff
@@ -49,6 +49,14 @@ public static ColumnEditor editor() {
     int jdbcType();
 
     /**
+     * Get the underlying type for this column where
+     * jdbcType() itself is the generic Types.ARRAY.
+     *
+     * @return a type constant for the specific database
+     */
+    int underlyingType();
```

Comment:
Let's call it "component type", re-using the term established in the Java world for it?